tools/tcpbind.py

```diff
@@ -0,0 +1,463 @@
+#!/usr/bin/python
+#
+# tcpbind       Trace IPv4 and IPv6 binds()s.
+#               For Linux, uses BCC, eBPF. Embedded C.
+#
+# based on tcpconnect utility from Brendan Gregg's suite.
+#
+# USAGE: tcpbind [-h] [-t] [-E] [-p PID] [-P PORT [PORT ...]] [-w]
```

Comment:
The syntax "-P PORT [PORT ...]" does not match the example, should it be comma separated list?

Reply:
Thank you, it was a good catch, fixed